packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -13,23 +13,22 @@ $ds-c-inset-border-width: $spacer-half;
 
   // Choice label
   + label {
-    align-items: center;
+    align-items: flex-start;
+    cursor: pointer;
     display: flex;
+    flex-direction: column;
     flex-wrap: wrap;
     font-weight: $font-normal;
+    justify-content: center;
     margin: $spacer-1 0;
-    max-width: $measure-base;
+    max-width: fit-content;
```

Comment:
We want to be doing this, we use `$measure-base` here for a reason. See this utility and theme variable:
http://localhost:3000/utilities/measure/

Reply:
Prod:
<img width="1024" alt="Screen Shot 2020-10-13 at 3 12 39 PM" src="https://user-images.githubusercontent.com/5424713/95911032-8be81480-0d66-11eb-941d-ad834acb57a3.png">

This PR:
<img width="1024" alt="Screen Shot 2020-10-13 at 3 13 54 PM" src="https://user-images.githubusercontent.com/5424713/95911160-b89c2c00-0d66-11eb-8e4a-fecffda39180.png">